Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -29,45 +30,73 @@ def getPatientUIDByPatientName(name):
   return None
 
 #------------------------------------------------------------------------------
-# Load patient by patient UID from DICOM database
 def loadPatientByUID(patientUID):
+  """ Load patient by patient UID from DICOM database
+  """
   if not hasattr(slicer,'dicomDatabase') or not hasattr(slicer.modules,'dicom'):
-    logging.error('DICOM module or database cannot be accessed!')
-    return
+    logging.error('DICOM module or database cannot be accessed')
+    return False
+
+  if not patientUID in slicer.dicomDatabase.patients():
+    logging.error('No patient found with DICOM database UID ' + str(patientUID))
```

Comment:
```
logging.error('No patient found with DICOM database UID %s' % patientUID)
```

Reply:
Here is a suggestion of improved message:
```
logging.error('Patient with UID [%s] not found in DICOM database' % patientUID)
```

The previous message seemed to indicated that the patient was not found the database having UID xyz

Reply:
Or if we want to have the UID at the end (to facilitate review of logs where multiple patient can not be found:

```
logging.error('No patient found with DICOM UID [%s]' % patientUID)
```

Reply:
The wording of the message is intentional to indicate that the UID is not DICOM PatientID, but DICOM database UID.